instrumentation/netty/netty-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_0/client/NettyClientSingletons.java

```diff
@@ -15,10 +15,19 @@
 
 public final class NettyClientSingletons {
 
-  private static final boolean alwaysCreateConnectSpan =
-      Config.get().getBoolean("otel.instrumentation.netty.always-create-connect-span", false);
-  private static final boolean sslTelemetryEnabled =
-      Config.get().getBoolean("otel.instrumentation.netty.ssl-telemetry.enabled", false);
+  private static final boolean connectionTelemetryEnabled;
+  private static final boolean sslTelemetryEnabled;
+
+  static {
+    Config config = Config.get();
+    boolean alwaysCreateConnectSpan =
+        config.getBoolean("otel.instrumentation.netty.always-create-connect-span", false);
```

Comment:
Should we try to communicate to users that they are using a deprecated property? Or will we just document the deprecation in release notes?

Reply:
Hmm, we've probably never done that before, but that's a good idea - @open-telemetry/java-instrumentation-approvers WDYT about adding a warning log if a deprecated property is used?